Packs/Cybereason/Scripts/CybereasonPreProcessingExample/CybereasonPreProcessingExample.py

```diff
@@ -0,0 +1,21 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+res = True
+incident = demisto.incidents()[0]
```

Comment:
@idovandijk , when running a pre-process script, do we have the current incident in our hans, meaning demisto.incident() should work?

Reply:
We would need to check in existing content, not sure